### ISSUE TITLE
Accessibility Audit: Bulk upload patient user flows

### ISSUE THREAD

**Issue by mehansen** [CDCgov/prime-simplereport#5781]
### Background
The SimpleReport team performs quarterly accessibility audits as a proactive approach to ensure our products comply with Section 508 of the Rehabilitation Act.

### Action requested
Use Axe DevTools to perform an accessibility audit of the bulk patient upload user flow. Do the page scan as well as any relevant guided tests.
Pages to visit:
* `/app/upload-patients`

Use cases to test:
* successful bulk upload (inc success toast)
* bulk upload produces validation messages
* bulk upload produces error toast

### Acceptance Criteria
Full audit completed and tickets created for any accessibility violations.

**Comment by BobanL** [comment]
Tickets created
- https://github.com/CDCgov/prime-simplereport/issues/5832
